client/cmd/dexc-desktop/pkg/build-windows.cmd

```diff
@@ -0,0 +1,59 @@
+@echo off
+
+:: This build script has been tested with Windows 10 and Windows 11.
+:: 
+:: See Build-Windows.md for more information.
+
+
+set buildDir=%cd%\build
+set libDir=%buildDir%\libs\webview2
+if not exist "%libDir%" (
+    mkdir %libDir%
+)
+
+set outputDir=%buildDir%\windows
+if exist "%outputDir%" (
+    rd /s /q %outputDir%
+)
+mkdir %outputDir%
+
+set exeFile=%outputDir%\dexc-desktop.exe
+
+set sitePath=..\..\webserver\site
+set workDir=%cd%
+
+:: If the site bundle does not exist, build it.
+if not exist "%sitePath%\dist" (
+    echo Building site bundle...
+    cd %sitePath%
+    call npm clean-install
+    call npm run build
+    cd %workDir%
+)
+
+echo Fetching and extracting the latest WebView2 SDK...
+curl -sSL "https://www.nuget.org/api/v2/package/Microsoft.Web.WebView2" | tar -xf - -C "%libDir%"
+
+if %ERRORLEVEL% NEQ 0 (
+    echo Fetching and extracting WebView2 failed! Do you have tar and curl?
+    exit /b 1
+)
+
+echo Building dexc-desktop with CGO configured...
+set CGO_CXXFLAGS="-I%libDir%\build\native\include"

```

Comment:
So, I changed this to exactly: `set CGO_CXXFLAGS="-I%libDir%\build\native\include" "-IC:\msys64\ucrt64\include"`

And it builds with a few warnings:
```
 .\pkg\build-windows.cmd
Building dexc-desktop with CGO configured...
# github.com/webview/webview
In file included from C:\msys64\ucrt64\include/winerror.h:4739,
                 from C:\msys64\ucrt64\include/winbase.h:2817,
                 from C:\msys64\ucrt64\include/windows.h:70,
                 from C:\msys64\ucrt64\include/rpc.h:16,
                 from C:\msys64\ucrt64\include/wtypesbase.h:7,
                 from C:\msys64\ucrt64\include/shlobj.h:9,
                 from webview.h:1051,
                 from webview.cc:1:
C:\msys64\ucrt64\include/fltwinerror.h:12: warning: "FACILITY_USERMODE_FILTER_MANAGER" redefined
   12 | #define FACILITY_USERMODE_FILTER_MANAGER 0x1f
      |
In file included from C:\msys64\ucrt64\include/winbase.h:2817,
                 from C:\msys64\ucrt64\include/windows.h:70,
                 from C:\msys64\ucrt64\include/rpc.h:16,
                 from C:\msys64\ucrt64\include/wtypesbase.h:7,
                 from C:\msys64\ucrt64\include/shlobj.h:9,
                 from webview.h:1051,
                 from webview.cc:1:
C:\msys64\ucrt64\include/winerror.h:38: note: this is the location of the previous definition
   38 | #define FACILITY_USERMODE_FILTER_MANAGER 31
      |
In file included from webview.h:1051,
                 from webview.cc:1:
C:\msys64\ucrt64\include/shlobj.h:607:3: warning: 'typedef' was ignored in this declaration
  607 |   typedef enum tagGPFIDL_FLAGS {
      |   ^~~~~~~
Build completed
        1 file(s) copied.
Signing C:\Users\joe\git\dcrdex\client\cmd\dexc-desktop\build\windows\dexc-desktop.exe
Usage: sign-windows.cmd <file>
The WebView2Loader.dll file should be included with dexc-desktop.exe
```

But it does build. Running is mostly fine, but The USDC has no symbol and the candles dont seem to load... Is it using an up to date site folder?

![image](https://github.com/decred/dcrdex/assets/34998433/c34c2afb-a3cc-4af3-bfe9-368cbd727abc)

Reply:
Oh yeah I updated my site folder and it worked. Maybe should do that anyway? I see it is conditional.

Can you add the include? I'm fine with the warnings it is building with that.

Reply:
> Can you add the include? I'm fine with the warnings it is building with that.

I tried hard but can't replicate this situation - the header files are all found correctly without adding this path, and the build runs without any warnings/errors.  I think there may be some leftover of the previous msys install on the system you're testing on.